late/chroot_scripts/03-ubuntu-drivers.sh

```diff
@@ -12,8 +12,17 @@ done
 if [ -n "$UBUNTU_DRIVERS_BLACKLIST" ]; then
     echo "UBUNTU_DRIVERS_BLACKLIST: $UBUNTU_DRIVERS_BLACKLIST"
 fi
+BIOSID=$(dmidecode -t 11 | sed '/ 1\[/!d; s,.* 1\[,,; s,\],,' | tr '[:lower:]' '[:upper:]')
```

Comment:
Isn't the kernel already doing this by providing`product_sku`? Why not use that instead?

Reply:
OK. Please review the commit again.